torch/autograd/grad_mode.py

```diff
@@ -60,11 +61,16 @@ class no_grad(_DecoratorContextManager):
         >>> z.requires_grad
         False
     """
+    def __init__(self):
+        if not torch._jit_internal.is_scripting():
+            super().__init__()
+        self.prev = False
+
     def __enter__(self):
         self.prev = torch.is_grad_enabled()
-        torch._C.set_grad_enabled(False)
+        torch.set_grad_enabled(False)
```

Comment:
@gchanan 

What do you think of these changes? In particular, what are the implications of changing `torch._C.set_grad_enabled` to `torch.set_grad_enabled`?

Reply:
These are fine, it's equivalent

Reply:
Actually this is the context manager below ;) that then calls the `_C` version.